tools/run_tests/jobset.py

```diff
@@ -177,6 +177,15 @@ def __cmp__(self, other):
     return self.identity() == other.identity()
 
 
+class JobResult(object):
+  def __init__(self):
+    self.state = "UNKNOWN"
```

Comment:
Nit: Please use single quotes for strings 'UNKNOWN' (here and throughout your changes).